fastlane/lib/fastlane/actions/changelog_from_git_commits.rb

```diff
@@ -37,6 +37,10 @@ def self.run(params)
         changelog = changelog.gsub("\n\n", "\n") if changelog # as there are duplicate newlines
         Actions.lane_context[SharedValues::FL_CHANGELOG] = changelog
 
+        puts ""
+        puts changelog
```

Comment:
I initially tried that, but because changelog is typically multiline, I think it would look weird:

```
[09:34:35]: • Commit message
• Commit message
• Commit message
• Commit message
```

I'm happy make that change though if you'd prefer it.